.travis.yml

```diff
@@ -6,10 +6,6 @@ addons:
 
 language: node_js
 node_js:
-  - "4"
-  - "6"
-  - "8"
-  - "9"
```

Comment:
We also need to check the code that parses this file and extracts the supported Node versions.

Reply:
Great catch, I'll check that